modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -58,6 +59,9 @@ def sort_by_recent_appointment(facilities)
       end
 
       def log_nil_cache
+        # not being hit because endpoint is not in use. might actually be helpful for indicating that something is wrong.
+        # hard to be sure without knowing more about how this would be used. if this endpoint were somehow called
+        # before appointments are fetched (which is plausible because pre-caching is turned off) this would just be normal behavior
         Rails.logger.info('mobile facilities info appointments cache nil', user_uuid: @current_user.uuid)
```

Comment:
Category: I am here with id.
Action: Leave for now because it's unclear to me why we're doing this. The code only attempts to fetch appointments from the cache, presumably on the assumption that they will have to be there. I don't know if that assumption is based on pre-caching or because the user has to navigate to appointments before this is relevant. Either way, it seems like this could indicate a problem. Since this code isn't in use, it's hard to be sure.